Web/src/app/app.config.ts

```diff
@@ -0,0 +1,8 @@
+import { Injectable } from '@angular/core';
+
+@Injectable()
+
+export class AppConfig {
```

Comment:
I'm thinking we could maybe use [Angular CLI Environments](https://tattoocoder.com/angular-cli-using-the-environment-option/) for this. I can add a feature for it and work on this

Reply:
Good thinking! I added a task in Teamwork for that